webrtc-c/webrtc-and-directstream/README.md

```diff
@@ -0,0 +1,63 @@
+# KVS WebRTC and Direct Stream Simultaneously
+## Overview
+
+Added a gstreamer app sample (`kvsWebRTCAndDirectStream.c`) to support streaming video to KVS with WebRTC SDK (for real-time use) and Stream Producer SDK (for near-realtime video ingestion), simultaneously from 1 camera source.
+
+This sample is only tested on Raspberry Pi equipped with USB Camera. 
+
+## Prerequisites
+
+- AWS Account with configured:
+  - Kinesis Video Streams
+  - IAM role with appropriate permissions
+
+## Installation and Configuration
+
+1. Clone this repository to your Raspberry Pi with submodules:
+   ```
+   git clone --recurse-submodules <repository-url>
```

Comment:
https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git
cd amazon-kinesis-video-streams-demos/webrtc-c/webrtc-and-directstream